common/azure_auth.go

```diff
@@ -191,7 +212,11 @@ func (aa *AzureAuth) acquirePAT(
 	if aa.temporaryPat != nil {
 		return aa.temporaryPat, nil
 	}
-	management, err := factory(azure.PublicCloud.ServiceManagementEndpoint)
+	env, err := aa.getAzureEnvironment()
+	if err != nil {
+		return nil, err
```

Comment:
please cover this one with test - e.g. call `aa. acquirePAT(nil)` with invalid azure env name